### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by cae67** [albermax/innvestigate#314]
Hi all,

Your assistance would be greatly appreciated on this question.

Can Innvestigate be directly applied to a Functional API-based Keras model with two parallel data inputs/streams that have multiple layers and that merge prior to a couple of final dense layers and a softmax output layer?

If that is not possible, is there another way to apply LRP to the network? I know that it is possible to split a functional model of that nature into 3 separate Sequential models (e.g., Model A - stream 1 model, Model B - stream 2 model, Model C - merger model) where LRP could be applied on Model C. However, I'm not sure if there is a way to take the relevance output from Model C and feed it into separate analyzers for Models A and B.

Thank you,
Charles
